backend/plugin/mfa/otp/otp.go

```diff
@@ -0,0 +1,102 @@
+// Package otp is the plugin for OTP Multi-Factor Authentication.
+package otp
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pquerna/otp/totp"
+	"golang.org/x/exp/slices"
+)
+
+var (
+	// issuerName is the name of the issuer of the OTP token.
+	issuerName = "Bytebase"
+	// secondsInMinute is the number of seconds in a minute.
```

Comment:
We should comment and declare variables to make the reader understand the behavior.

In particular, it's not we check past 5 secrets, instead, it's that the secret is valid for 5 minutes.